shared/scss/base/_reset.scss

```diff
@@ -25,7 +25,6 @@ picture,
 svg,
 video {
   display: block;
-  width: 100%;
```

Comment:
this file is currently active in our projects. 
I think if this line of codes affect on some of your code you can add some changes in your own code to disable it in your code.
what do you think?